### PR TITLE
Adding Desktop to System.ComponentModel.TypeConverter

### PR DESCRIPTION
**Pull request by venkat-raman251**

### CODE REVIEW

**Comment by venkat-raman251** [comment]
@weshaggard : Please have a look

File: src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj [hunk 1]
```diff
@@ -9,14 +9,14 @@
     <RootNamespace>System.ComponentModel.TypeConverter</RootNamespace>
     <AssemblyName>System.ComponentModel.TypeConverter</AssemblyName>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
-    <PackageTargetFramework>dotnet5.1</PackageTargetFramework>
```
Comment (weshaggard) [inline src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]: As with the others you need to make this conditional and net45 needs to have IsPartialFacadeAssembly=true

Comment (venkat-raman251) [inline src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]: Committed with changes.

File: src/System.ComponentModel.TypeConverter/src/project.json [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
-    "System.Reflection.Extensions": "4.0.0",
-    "System.Reflection.Primitives": "4.0.0",
-    "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.0",
-    "System.Runtime.Extensions": "4.0.0",
-    "System.Threading": "4.0.0"
-  },
   "frameworks": {
-    "dnxcore50": {}
+    "dnxcore50": {
```
Comment (weshaggard) [inline src/System.ComponentModel.TypeConverter/src/project.json]: Please update dnxcore50 to dotnet5.1

**Comment by venkat-raman251** [comment]
@weshaggard : Updated the project.json

**Comment by weshaggard** [comment]
LGTM merge when CI is green.

**Comment by weshaggard** [comment]
@dotnet-bot test  Innerloop OpenSUSE13.2 Debug Build and Test

**Pull request merged**